app/views/list/new.html.slim

```diff
@@ -0,0 +1,18 @@
+.container-fluid
+  .row
+    .col-xs-12
+      h1 = "Create Wishlist"
+      - if flash[:error]
```

Comment:
you should be using the model's validation errors here
http://guides.rubyonrails.org/active_record_validations.html#displaying-validation-errors-in-views